docs/architecture/adr-005-qgb-reduce-state-usage.md

```diff
@@ -1,5 +1,9 @@
 # ADR 005: QGB Reduce State Usage
 
+## Status
+
+Proposed
```

Comment:
@SweeXordious is it safe to update this to Accepted or Implemented? I'm not confident because the `## Decision` section has incomplete checkboxes

Reply:
`This one is huum, I implemented an MVP for it in [here](https://github.com/sweexordious/celestia-app/tree/qgb_working_draft_of_new_ingestion_2.0) but we never really had a deployed version. Instead, we stopped working on it and switched to [this](https://github.com/celestiaorg/orchestrator-relayer/pull/66). 

Can we put in the status something like `replaced by deciding to use a P2P network` or similar?

Reply:
Yea, we have a Deprecated option. Is there a doc I can link to for the P2P network?
```suggestion
Deprecated
```

Reply:
It's this PR: https://github.com/celestiaorg/orchestrator-relayer/pull/66 that contains the ADr, which is still not merged

Reply:
Resolved by f34bc88914e437c58aafb1f2039d2da11c308071